src/native/libs/System.Globalization.Native/pal_collation.m

```diff
@@ -64,6 +64,9 @@ static NSStringCompareOptions ConvertFromCompareOptionsToNSStringCompareOptions(
     if (comparisonOptions & IgnoreWidth)
         options |= NSWidthInsensitiveSearch;
 
+    if (comparisonOptions & IgnoreKanaType)
+        options |= NSCaseInsensitiveSearch | NSDiacriticInsensitiveSearch | NSWidthInsensitiveSearch;
```

Comment:
this doesn't look correct to me, ignoring Kana looks more complicated and we have custom code in e.g. FillIgnoreKanaRules.

Reply:
I created separate PR for IgnoreKanaType changes https://github.com/dotnet/runtime/pull/96002, please take a look. Once it will be merged will bring that changes to this PR also.